frontend/src/routes/Infrastructure/InfraEnvironments/Details/InfraEnvironmentDetailsPage.css

```diff
@@ -0,0 +1,8 @@
+.pf-c-dropdown__menu {
+    width: 270px !important;
+}
+
+.dropdownitem_title_bmc {
```

Comment:
Similarly here, IMHO we should stick with PatternFly defined style for the Group title